src/renderer/js/component/shapeShift/internal/active-shift.jsx

```diff
@@ -0,0 +1,161 @@
+// @flow
+import * as React from "react";
+import QRCode from "qrcode.react";
+import * as statuses from "constants/shape_shift";
+import Address from "component/address";
+import Link from "component/link";
+import type { Dispatch } from "redux/actions/shape_shift";
+import ShiftMarketInfo from "./market_info";
+
+type Props = {
+  shiftState: ?string,
+  shiftCoinType: ?string,
+  shiftDepositAddress: ?string,
+  shiftReturnAddress: ?string,
+  shiftOrderId: ?string,
+  originCoinDepositMax: ?number,
+  clearShapeShift: Dispatch,
+  doShowSnackBar: Dispatch,
+  getActiveShift: Dispatch,
+  shapeShiftRate: ?number,
+  originCoinDepositMax: ?number,
+  originCoinDepositFee: ?number,
+  originCoinDepositMin: ?string,
+};
+
+class ActiveShapeShift extends React.PureComponent<Props> {
+  constructor() {
+    super();
+    // $FlowFixMe
+    this.continousFetch = undefined;
```

Comment:
outside of the constructor, I think you need to add

`continuousFetch: ?number;`

as per https://flow.org/en/docs/types/classes/

Reply:
this should get rid of all the flow fix mes